server/src/main/java/org/elasticsearch/common/Rounding.java

```diff
@@ -61,7 +61,8 @@ public enum DateTimeUnit {
 
             @Override
             long roundFloor(long utcMillis, int multiplier) {
-                return DateUtils.roundWeekIntervalOfWeekYear(utcMillis, multiplier);
+                assert multiplier == 1;
```

Comment:
Not sure about this change. If we want the change as a short term one, to profile the outcome in an integrated system, it's OK, but otherwise we would better change the API.

Reply:
I'm not sure either. 
Yeah, that's the intention